### ISSUE TITLE
error with class properties

### ISSUE THREAD

**Issue by Nabrok** [babel/eslint-plugin-babel#142]
When using 5.0.0 on a react component with class properties I get `context.getSourceCode is not a function`.

Reverting back to 4.1.2 works.

**Comment by existentialism** [comment]
@Nabrok thanks for the report. I haven't been able to reproduce this, can you perhaps provide a small repo that reproduces the issue?

**Comment by Nabrok** [comment]
Here's a simple example.

```
import React from 'react';
import { render } from 'react-dom';

class App extends React.Component {
        state = {
                count: 0
        };

        _increment = () => this.setState({ count: this.state.count + 1 });

        render() {
                return <input type="button" onClick={this._increment} value={this.state.count} />;
        }
}

render(<App/>, document.getElementById('app'));
```
.eslintrc.yml ...
```
env:
  browser: true
  es6: true
extends:
  - 'eslint:recommended'
  - 'plugin:react/recommended'
parser: "babel-eslint"
parserOptions:
  ecmaFeatures:
    experimentalObjectRestSpread: true
    jsx: true
  sourceType: module
plugins:
  - react
  - babel
rules:
  indent:
    - error
    - tab
    - {
      SwitchCase: 1
    }
  linebreak-style:
    - error
    - unix
  quotes:
    - off
    - double
  semi:
    - off
  babel/semi:
    - error
    - always
    - {
      omitLastInOneLineBlock: true
    }
```
package.json
```
{
  "name": "eslint_test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "babel-eslint": "^8.2.3",
    "eslint": "^4.19.1",
    "eslint-plugin-babel": "^5.0.0",
    "eslint-plugin-react": "^7.7.0",
    "react": "^16.3.2"
  }
}
```

I thought it might be the arrow function, so I tried it without that but still the same problem.

**Comment by existentialism** [comment]
@Nabrok ooh, it's with the `omitLastInOneLineBlock` option! Just pushed a fix -> https://github.com/babel/eslint-plugin-babel/pull/144!

**Comment by existentialism** [comment]
@Nabrok fix released in 5.1.0!
